compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangEqualsExpr.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year 2018?

Reply:
BLangEqualsExpr.java class is no longer available.